core/lib/spree/app_configuration.rb

```diff
@@ -587,5 +587,11 @@ def admin_vat_location
         country: Spree::Country.find_by(iso: admin_vat_country_iso)
       )
     end
+
+    def active_storage_enabled?
+      @active_storage_enabled ||=
+        Spree::Config.image_attachment_module == Spree::Image::ActiveStorageAttachment ||
+        Spree::Config.taxon_attachment_module == Spree::Taxon::ActiveStorageAttachment
+    end
```

Comment:
I don't think it would make sense to see references to `Spree::Image::ActiveStorageAttachment` and `Spree::Taxon::ActiveStorageAttachment` classes before they are defined, I would feel completely lost. `Add check on Rails version if ActiveStorage enabled` introduces an additional check that builds on the code present on the this first commit.

Reply:
`Set ActiveStorage host in base controllers` commit should be after `Add check on Rails version if ActiveStorage enabled` 👍